Sources/Latte+WKWebView.swift

```diff
@@ -65,7 +65,6 @@ class LatteWKWebView: WKWebView, WKNavigationDelegate {
         let userScript = WKUserScript(source: initScript, injectionTime: .atDocumentStart, forMainFrameOnly: false)
         self.configuration.userContentController.addUserScript(userScript)
         self.configuration.userContentController.add(MessageHandler(self), name: "latteEvent")
-        self.configuration.processPool.perform(Selector(("_setCookieAcceptPolicy:")), with: HTTPCookie.AcceptPolicy.always)
```

Comment:
I guess we still need to set it, but with another approach?

- https://developer.apple.com/documentation/webkit/wkwebview
- https://developer.apple.com/documentation/webkit/wkwebview/1414979-configuration
- https://developer.apple.com/documentation/webkit/wkwebviewconfiguration
- https://developer.apple.com/documentation/webkit/wkwebviewconfiguration/1395661-websitedatastore
- https://developer.apple.com/documentation/webkit/wkwebsitedatastore
- https://developer.apple.com/documentation/webkit/wkwebsitedatastore/2881956-httpcookiestore
- https://developer.apple.com/documentation/webkit/wkhttpcookiestore
- https://developer.apple.com/documentation/webkit/wkhttpcookiestore/4183559-setcookiepolicy

Reply:
@louischan-oursky The api is only available in ios 17. And I've checked that using the private api `_setCookieAcceptPolicy` may cause the app rejected by app store. Therefore it seems we don't have a way to set that in <= ios 16.

Reply:
https://bugs.webkit.org/show_bug.cgi?id=140205
It seems the default policy already allows first party cookies, so I guess we don't have to set it.